src/api/quantum/meta/Universal.java

```diff
@@ -0,0 +1,13 @@
+package api.quantum.meta;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Represents a universal object
+ */
+@Target({ElementType.TYPE, ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.PACKAGE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Universal {}
```

Comment:
In that case it might do better to just create SideOnly annotation that supports listing, maybe call it something like SidesList or something